pom.xml

```diff
@@ -1759,7 +1774,6 @@
                       <exclude>commons-logging:commons-logging</exclude>
                       <!-- Use org.glassfish.hk2.external:jakarta.inject -->
```

Comment:
this comment seems not needed too.

Reply:
It is needed. We want to exclude `javax.inject:javax.inject` and use `org.glassfish.hk2.external:jakarta.inject`. They are sharing the same namespace (surprise!)